docs/resources/azurerm_network_interface.md.erb

```diff
@@ -0,0 +1,127 @@
+---
+title: About the azurerm_network_interface Resource
+platform: azure
+---
+
+# azurerm\_network\_interface
+
+Use the `azurerm_network_interface` InSpec audit resource to test properties and configuration of Azure Network Interface.
+<br />
+
+## Azure REST API version
+This resource interacts with version `2018-11-01` of the Azure Management API. For more
+information see the [Official Azure Documentation](https://docs.microsoft.com/en-us/rest/api/virtualnetwork/networkinterface(preview)/get).
+
+At the moment, there doesn't appear to be a way to select the version of the
+Azure API docs. If you notice a newer version being referenced in the official
+documentation please open an issue or submit a pull request using the updated
+version.
+
+## Availability
+
+### Installation
+
+This resource is available in the `inspec-azure` [resource
+pack](https://www.inspec.io/docs/reference/glossary/#resource-pack). To use it, add the
+following to your `inspec.yml` in your top-level profile:
+
+    depends:
+      inspec-azure:
+        git: https://github.com/inspec/inspec-azure.git
+
+You'll also need to setup your Azure credentials; see the resource pack
+[README](https://github.com/inspec/inspec-azure#inspec-for-azure).
+
+### Version
+
+This resource first became available in 1.7.0 of the inspec-azure resource pack.
```

Comment:
this should be targeted for 1.7.2 as the resource packed is currently at 1.7.1. Please update any instance of this is the other files.

Reply:
resolved